.circleci/config.yml

```diff
@@ -827,13 +827,14 @@ jobs:
 
   b_ubu_static:
     # On large runs 2x faster than on medium. 3x on xlarge.
-    <<: *base_ubuntu2204_xlarge
+    <<: *base_ubuntu2004_xlarge
     environment:
       TERM: xterm
       MAKEFLAGS: -j 10
       CMAKE_OPTIONS: -DCMAKE_BUILD_TYPE=Release -DUSE_Z3_DLOPEN=ON -DUSE_CVC4=OFF -DSOLC_STATIC_STDLIBS=ON
+      FORCE_RELEASE: ON
     steps:
-      - checkout
+      - run: git clone -b "v0.8.18" https://github.com/ethereum/solidity.git .
```

Comment:
Ideally we'd do a manual bytecode comparison against the released binary - that'd give full confidence.

Reply:
Too much hassle to setup via CI, but just do what ``prepare_bytecode_report:`` in the CI config does once with the old once with the new binary and compare the reports - if they match, we're good.

Reply:
> You should base this branch on the `v0.8.18` tag instead of current `develop`.
> 
> This might work but with the tag you'd be sure that you won't get 0.8.19 somewhere in there by accident.

I changed to what you suggested but a nightly was built instead: https://app.circleci.com/pipelines/github/ethereum/solidity/28365/workflows/5dede3e8-1f98-4e39-9929-ab0feb55d4b6/jobs/1258917/artifacts

Reply:
Yeah? Looks like a release to me now...

Reply:
oh no, actually it is correct. I looked at the wrong job haha

Reply:
Actually, doesn't :-D.

Reply:
Oh, you removed the forced checkout? No, you need to keep that - I think @cameel just needs to *additionally* base the branch on the release tag, but that alone will not be enough, you really have to force checkout the tag here.

Reply:
But it should be fine now, if you just add this line back.

Reply:
haha right.

Reply:
> I think @cameel just meant to additionally base the branch on the release tag, but that alone will not be enough, you really have to force checkout the tag here. No other way to preserve the commit hash and we need that.

Yeah, exactly. I meant not just removing the line but replacing it with **having the commit based on the tag**. That second part is very important. And in addition to that you still need `FORCE_RELEASE` and a custom SHA of course, because adding any commit (or even uncommitted changes) on top of the tag changes the commit hash used in the version.

Reply:
> But it should be fine now, if you just add this line back.

Well, I'm suggesting changing the base commit because to me it's less of a wildcard than having a frankenbuild that based on 0.8.19 that just checks out 0.8.18. Harder to think where things can go wrong with that. Changing base just seems much simpler to me.

Reply:
> Ideally we'd do a manual bytecode comparison against the released binary - that'd give full confidence.

Well, we will have a bytecode run here and also a PR to solc-bin will run comparison with binaries that are already there (except Windows).

So it's probably safe anyway but just trying to guard against unknowns. There's always something unexpected that pops up.

Reply:
ok, after discussion on Matrix, I realized that your solution does no require force changing the SHA so it's better if it works. Disregard my suggestion.